test/functional/testlib.bash

```diff
@@ -1229,6 +1229,50 @@ sign_manifest() { # swupd_function
     -outform DER -out "$(dirname "$manifest")"/Manifest.MoM.sig
 }
 
+# Signs the latest_versin with a PEM key and generates the signed latest_version in the same location
+# Parameters:
+# - latest_version: the path to the latest_version to be signed
+sign_latest_version() {
+	local latest_version=$1
+
+	# If no parameters are received show usage
+	if [ $# -eq 0 ]; then
+		cat <<-EOM
+			Usage:
+			    sign_latest_version <latest_version>
+			EOM
+		return
+	fi
+	validate_item "$latest_version"
+
+	sudo openssl smime -sign -binary -in "$latest_version" \
+		-signer "$FUNC_DIR"/Swupd_Root.pem \
+		-inkey "$FUNC_DIR"/private.pem \
+		-outform DER -out "$(dirname "$latest_version")"/latest_version.sig
+}
```

Comment:
this is a picky request, but please leave an empty line between the opening bracket and the closing bracket. All other functions have it, so it looks consistent and cleaner.
```
my_function() {

# some code
# more code

}
```

Reply:
Done